mender-convert-modify

```diff
@@ -113,11 +114,15 @@ if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
             log_info "Installing Mender Auth and Mender Update version ${MENDER_CLIENT_VERSION}"
             deb_get_and_install_package mender-auth "${MENDER_CLIENT_VERSION}"
             deb_get_and_install_package mender-update "${MENDER_CLIENT_VERSION}"
+            mender_client_deb_name="${DEB_NAME}"
+
+            log_info "Installing Mender Flash version ${MENDER_FLASH_VERSION}"
+            deb_get_and_install_package mender-flash "${MENDER_FLASH_VERSION}"
             ;;
     esac
 
     # Save installed client version for tests in Yocto variable format
-    testscfg_add "PREFERRED_VERSION:mender-client" "$(echo ${DEB_NAME} | sed -r 's/.*_([0-9]+\.[0-9]+\.[0-9]+).*/\1/')"
+    testscfg_add "PREFERRED_VERSION:mender-client" "$(echo ${mender_client_deb_name} | sed -r 's/.*_([0-9]+\.[0-9]+\.[0-9]+).*/\1/')"
```

Comment:
In both cases you are just setting `mender_client_deb_name` to `DEB_NAME`. So why introduce it?

Reply:
Not very intuitive, I know.

`DEB_NAME` is defined in `deb_get_and_install_package` as a way for the user too know the file name that was actually installed. When installing the new client, we do install later `mender-flash` and then `DEB_NAME` gets overridden.

If you want, I can install first `mender-flash` and then the client packages and hence leave here `DEB_NAME`. Although that approach will make it weaker for future changes.

Reply:
No, it's fine, it just looked like it was done for no reason, and hence I suspected it was a leftover.